playground/backend/containers/go/Dockerfile

```diff
@@ -63,10 +63,27 @@ RUN mkdir /opt/mitmproxy &&\
     wget https://snapshots.mitmproxy.org/7.0.4/mitmproxy-7.0.4-linux.tar.gz &&\
     tar -zxvf mitmproxy-7.0.4-linux.tar.gz &&\
     mkdir /usr/local/share/ca-certificates/extra
-COPY entrypoint.sh /
 COPY allow_list_proxy.py /opt/mitmproxy/
 COPY allow_list.py /opt/mitmproxy/
 ENV HTTP_PROXY="http://127.0.0.1:8081"
 ENV HTTPS_PROXY="http://127.0.0.1:8081"
 
-ENTRYPOINT ["/entrypoint.sh"]
+COPY entrypoint.sh /
+COPY proxy.sh /
+# Run proxy
+RUN chmod +x /proxy.sh
+RUN /proxy.sh
+
+# Create a user group userGroup
```

Comment:
Isn't it called `appuser` instead of `userGroup`?

Reply:
Changed.